odoo/addons/base/migrations/12.0.1.3/pre-migration.py

```diff
@@ -0,0 +1,69 @@
+# © 2018 Opener B.V. (stefan@opener.amsterdam)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.addons.openupgrade_records.lib import apriori
+from openupgradelib import openupgrade
+
+xmlid_renames = [
+    ('auth_signup.default_template_user', 'base.template_portal_user_id'),
+]
```

Comment:
why not also add this?
```python
('auth_signup.default_template_user_config', 'base.default_template_user_config')
```

Reply:
I think the module merge should take care of this. This is not the case for the user as the name part of the XML id is different.

Reply:
Sorry, you are right. This module is not actually merged into base.